src/Components/CreateImageWizard/formComponents/Oscap.tsx

```diff
@@ -135,9 +183,12 @@ const ProfileSelector = ({ input }) => {
         isDisabled={!isSuccess}
         onFilter={(_event, value) => {
           return [
-            <OScapNoneOption setProfileName={setProfileName} key={profiles} />,
+            <OScapNoneOption
+              setProfileName={setProfileName}
+              key="oscap-none-option"
+            />,
           ].concat(
-            profiles.map((profile_id, index) => {
+            (profiles || []).map((profile_id, index) => {
```

Comment:
That's a pretty original way of solving this!
Have you thought about making the profile not null if empty in the first step instead?

Reply:
I define it with new variable const safeProfiles = useMemo(() => profiles || [], [profiles]);
but I am not sure why is it better then  it is now